kernelci.toml.sample

```diff
@@ -17,20 +18,23 @@
 
 # -- Sample test lab --
 #
-["lab:my-lava-lab"]
+# [lab.my-lava-lab]
 #
 # user = "user-name"
 # lab_token = "1234-5678"
 
 # -- Sample database --
 #
-["db:localhost"]
+[db.localhost]
 #
 # db_token = "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX"
 # api = "http://192.168.122.1:5001"
 # callback_id = "kernelci-callback-local"
 # callback_url = "http://localhost:5001"
 
+# [db."staging.kernelci.org"]
```

Comment:
I think we can also just drop the convention of having config entry names match the FQDN.  We could just have `staging` in the YAML config or any arbitrary string as long as there's no ambiguity.

Reply:
Yes, that works too. I'll keep this line in the sample though for the reference: `Make sure to surround sub-section name with "" (double-quotes) if it contains "." (dot) in the name e.g [db."staging.kernelci.org"].`

Reply:
I'll send a PR for the patch file to update staging settings file with dotted keys.